client/src/dataProviders/profileDataProviders.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+import {dispatchReceivedData} from './dataProvidersUtils'
+import {DEFAULT_PROVIDER_KEEP_ALIVE} from '../constants'
+import {loadImageAsync, mapArrayToId, mapObjToId} from '../utils'
+
+export const politiciansProvider = () => ({
+  ref: 'politicians',
+  getData: [
+    fetch,
+    `${process.env.REACT_APP_API_URL || ''}/api/k/list_politicians?cachebreak`,
+    {
+      accept: 'application/json',
+    },
+  ],
+  onData: [dispatchReceivedData, ['profile', 'list']],
+  keepAliveFor: DEFAULT_PROVIDER_KEEP_ALIVE,
+})
+
+export const cadastralInfoProvider = (id: string, needed: boolean = true) => ({
+  ref: `politician-cadastral-info-${id}`,
+  getData: [
+    fetch,
+    `${process.env.REACT_APP_API_URL || ''}/api/k/kataster_info_politician?id=${id}`,
+    {
+      accept: 'application/json',
+    },
+  ],
+  onData: [dispatchReceivedData, ['profile', 'cadastral'], mapArrayToId, id, 'parcelno'],
+  keepAliveFor: DEFAULT_PROVIDER_KEEP_ALIVE,
+  needed,
+})
+
+export const detailsProvider = (id: string) => ({
+  ref: `politician-details-${id}`,
+  getData: [
+    fetch,
+    `${process.env.REACT_APP_API_URL || ''}/api/k/info_politician?id=${id}`,
```

Comment:
Why we don't use `dev.production` for this constant too?

Reply:
Not sure I know what you mean. This is leaving possibility for configuration wherever the api is (in case the app would on some different subdomain).

Reply:
REACT_APP_API_URL is defined in dev.development file. We use || to cover the production environment. Why not define dev.production and avoid the logical or in every use of the constant?

Reply:
I'd rather have this than having to define empty env var for when the api is on the same url, or debugging fetches for `undefined/api/stuff`.

It also makes sense as having the reasonable default of empty string (server lives on the same place as the rest of the app), which you can optionally override using env var.